format/mbr/mbr.go

```diff
@@ -0,0 +1,66 @@
+package mbr
+
+import (
+	"fmt"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/registry"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+func init() {
+	registry.MustRegister(decode.Format{
+		Name:        format.MBR,
+		Description: "Master Boot Record",
+		DecodeFn:    mbrDecode,
+	})
+}
+
+func decodePartitionTableEntry(d *decode.D) {
+	d.FieldU8("boot_indicator", scalar.UToDescription{
+		0x80: "active",
+		0x00: "inactive",
```

Comment:
You can do a bit field struct here (struct with `d.FieldBool("...")`) if it's useful to be able to address individual bits. Sometimes i also use `0b1000_0000` etc literals for binary literals

Reply:
Maybe this could use `scalar.UToSym` that way you can write queries as `.boot_indicator  == "active"` and if you really want to compare the non-mapped value do `(.boot_indicator | toactual) == 0x80`